crates/bevy_picking/src/backend.rs

```diff
@@ -84,7 +84,7 @@ pub struct PointerHits {
 }
 
 impl PointerHits {
-    #[allow(missing_docs)]
+    #[expect(missing_docs)]
```

Comment:
missing reason

Reply:
IMO we should just add trivial docs to these methods.